docs/config/index.md

```diff
@@ -366,21 +393,21 @@ Listen to port and serve API. When set to true, the default port is 55555
 - **Type:** `boolean`
 - **Default:** `false`
 
-Will call `.mockClear()` on all spies before each test
+Will call [`.mockClear()`](/api/#mockclear) on all spies before each test. This will clear mocks history, but not reset its implementation to default one.
 
 ### mockReset
 
 - **Type:** `boolean`
 - **Default:** `false`
 
-Will call `.mockReset()` on all spies before each test
+Will call [`.mockReset()`](/api/#mockreset) on all spies before each test. This will clear mocks history and reset its implementation to an empty function (will return `undefined`).
 
 ### restoreMocks
 
 - **Type:** `boolean`
 - **Default:** `false`
 
-Will call `.mockRestore()` on all spies before each test
+Will call [`.mockRestore()`](/api/#mockrestore) on all spies before each test. This will clear mocks history and reset its implementation to the original one.
```

Comment:
```suggestion
Will call [`.mockRestore()`](/api/#mockrestore) on all spies before each test. This will clear mock history and reset its implementation to the original one.
```